apps/stripe/src/__tests__/mocks/constants.ts

```diff
@@ -11,3 +11,4 @@ export const mockAdyenWebhookUrl = `${mockAppUrlBase}?${new URLSearchParams({
   [WebhookParams.saleorApiUrlSearchParam]: mockedSaleorApiUrl,
   [WebhookParams.configurationIdIdSearchParam]: mockedConfigurationId,
 }).toString()}`;
+export const mockedSaleorTransactionId = btoa("Transaction:1");
```

Comment:
I just checked and Saleor is using `TransactionItem:32a289ca-93f5-44ec-a0b2-fb4063eb2119` under the hood - also we can maybe use the same pattern for other mocked data coming from Saleor?

Reply:
Yea, but TBH I think I will just mock it, because it raises confusion. It's impelemntation detail what saleor does and we should not rely on that